python/pythonmonkey/builtin_modules/console.js

```diff
@@ -47,9 +47,9 @@ class Console {
         ignoreErrors = true;
       options = { stdout, stderr, ignoreErrors };
     }
-
-    this.#writeToStdout = options.stdout.write;
-    this.#writeToStderr = options.stderr.write;
+    /* s.replace workaround for https://github.com/Distributive-Network/PythonMonkey/issues/89 */
+    this.#writeToStdout = (s) => options.stdout.write(s.replace(/[\uD800-\uDFFF]/g, '\uFFFD'));
+    this.#writeToStderr = (s) => options.stderr.write(s.replace(/[\uD800-\uDFFF]/g, '\uFFFD'));
```

Comment:
No longer required since we have https://github.com/Distributive-Network/PythonMonkey/pull/98